src/main/webapp/app/shared/layouts/footer/footer.component.ts

```diff
@@ -14,6 +14,12 @@ export class FooterComponent implements OnInit {
     readonly requestChangeUrl = 'https://github.com/ls1intum/Artemis/issues/new/choose';
 
     email: string;
+    gitBranch: string;
+    gitCommitId: string;
+    gitTimestamp: string;
+    gitCommitUser: string;
+    testServer: Boolean;
+    inProduction: Boolean;
```

Comment:
```suggestion
    inProduction: boolean;
```

Reply:
It is recommended to always use primitive boolean type for declarations: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#the-primitives-string-number-and-boolean